### DISCUSSION TITLE
minimum vram required?

### DISCUSSION THREAD

**Discussion by douchebanner** [volotat/SD-CN-Animation#100] [Q&A] [answered]
whats the minimum vram required?

has anyone run this with 6 gb?

**Comment by volotat** [comment, marked as answer]
Not sure about minimum requirements, as it is very dependent on the size of the processing video, but I could say that to reproduce my results  it would require about 8 gb of vram as it is very beneficial to have at least HD sized images at the processing stage.

**Comment by matthewdeacon74** [comment]
Late response - but I've been using it on a laptop with a 6GB RTX2060 because that's the only nVidia card I have right now.  I can reliably do 1280x720 videos, or 1024x1024 if I reedit the source vid to be square. 
It's not fast; roughly 33s per frame at these sizes, so 300 frames (10s @ 30fps) takes about 3 hours.
